### PR TITLE
[dom] Update Extrae to TC 19.03

### PR DESCRIPTION
**Pull request by kraushm**

### CODE REVIEW

**Comment by kraushm** [comment]
retest this please

**Comment by kraushm** [comment]
retest this please

File: easybuild/easyconfigs/d/Dimemas/Dimemas-5.3.6.eb [hunk 1]
```diff
@@ -11,7 +11,7 @@ toolchain = {'name': 'dummy', 'version': ''}
 source_urls = ['https://ftp.tools.bsc.es/dimemas']
 sources = ['dimemas-%(version)s-Linux_x86_64.tar.bz2']
```
Comment (jgphpc) [inline easybuild/easyconfigs/d/Dimemas/Dimemas-5.3.6.eb]: version = '5.4.1'

File: easybuild/easyconfigs/c/clusteringsuite/clusteringsuite-2.6.7.eb [hunk 2]
```diff
@@ -10,7 +10,7 @@ applications' computation structure."""
 toolchain = {'name': 'dummy', 'version': ''}
 source_urls = ['http://ftp.tools.bsc.es/clusteringsuite/']
 sources = ['%(name)s-%(version)s-Linux_x86_64.tar.bz2']
-checksums = ['9ea5f737616ec7c342f68eb1c2c31225']
```
Comment (jgphpc) [inline easybuild/easyconfigs/c/clusteringsuite/clusteringsuite-2.6.7.eb]: version = '2.6.8'

**Comment by jgphpc** [comment]
:+1: 
need to update 2 versions

**Pull request merged**